README.md

```diff
@@ -109,6 +109,13 @@ scrape_configs:
         replacement: 127.0.0.1:9115  # Blackbox exporter.
 ```
 
+## Permissions
+
+The ICMP probe requires elevated privileges to function:
+
+* *Linux*: root user _or_ `CAP_NET_RAW` capability is required.
+  * Can be set by executing `setcap cap_net_raw+ep blackbox_exporter`
+* *BSD / OS X*: root user is required.
```

Comment:
I guess it's the same on Windows too, or should we just leave it unspecified?

We could just say "Other Unix", which includes Windows technically :)